src/ldw-barsPlus.js

```diff
@@ -700,8 +700,20 @@ export default {
           }
         }
       })
-      .on("mouseenter", function (d) {
-        if (g.inSelections || g.editMode) return;
+      .on("touchstart", function(d){ //WIP toching should NOT give the hover effect,, waiting for a proper testing device to continue
+        d3.select(this)
+          .style("opacity", "1.0")
+          .attr("stroke", "none")
+        ;
+      })
+      .on("touchend", function(d){ //WIP toching should NOT give the hover effect,, waiting for a proper testing device to continue
+        d3.select(this)
+          .style("opacity", "1.0")
+          .attr("stroke", "none")
+        ;
+      })
+      .on("mouseenter", function (d,e) {
+        if (g.editMode) return;
```

Comment:
Would be nicer to add something here that checks if touch is enabled or not. Doing the above just adds more complexity. On an actual device none of this might not be needed, so I'd rather we be on the safe side (and not add this code)